### PR TITLE
Support Scala 3 on Scala.js

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

File: build.sc [hunk 1]
```diff
@@ -6,8 +6,9 @@ val scalaVersions = "2.11.12" :: "2.12.13" :: "2.13.4" :: "3.0.0-RC1" :: dottyVe
 val scala2Versions = scalaVersions.filter(_.startsWith("2."))
 
 val scalaJSVersions = for {
-  scalaV <- scala2Versions
+  scalaV <- scalaVersions
   scalaJSV <- Seq("0.6.33", "1.4.0")
+  if scalaV.startsWith("2.") || scalaJSV.startsWith("1.")
```
Comment (jodersky) [inline build.sc:11]: just my two cents on DRYness: IMO, making the version tuples explicit here is much easier to understand and also to play around with when editing. We don't really gain much code reuse if we need to subsequently filter many combinations.

**Review by jodersky — APPROVED** [review]
LGTM! thanks for adding it

File: build.sc (hunk 1 above)
Comment (lolgab) [reply build.sc:11]: Yeah, when you have to filter out multiple things it becomes not nice.. I like that makes it harder to forget one combination.
But I think that we'll stop publishing for Scala.js eventually, removing this extra part.

Comment (jodersky) [reply build.sc:11]: no worries, just wanted to mention it because you mentioned DRY. It'd not at all a blocker

**Comment by jodersky** [comment]
@lolgab, what is your preferred way to proceed here. Should I merge and publish, or would you like to?

**Comment by lolgab** [comment]
@jodersky Whatever you prefer. Let's merge it

**Pull request merged**